android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties

```diff
@@ -322,8 +313,7 @@ Declare war on [civName]? = 向[civName]宣战吗？
 [leaderName] of [nation] = [nation]的[leaderName]
 You'll pay for this! = 你会为此付出代价的！
 Negotiate Peace = 议和
- # Requires translation!
-Peace with [civName]? = 
+Peace with [civName]? = 与 [civName] 签署和平条约？
```

Comment:
Chinese doesn't need the blank word, the blank is needed by other languages such as english, french and so on.

Reply:
> Chinese doesn't need the blank word, the blank is needed by other languages such as english, french and so on.

Oh,I know,I'm so sorry.